graphql/appointment/fields.ts

```diff
@@ -80,7 +80,7 @@ export class ExtendedFieldsLectureResolver {
         return isParticipant;
     }
     @FieldResolver((returns) => [AppointmentParticipant], { nullable: true })
-    @Authorized(Role.OWNER, Role.APPOINTMENT_PARTICIPANT, Role.ADMIN)
```

Comment:
@Fabian5150 I think you still need this one - I currently get

```
Requiring one of the following roles: OWNER,APPOINTMENT_PARTICIPANT,ADMIN
```

Reply:
Sorry I missed your comment here. John added it by now and it seems to work. :)